thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -307,13 +285,47 @@ def setup_distributed(self, model):
                     sharding_strategy=sharding_strategy,
                     bucketing_strategy=bucketing_strategy,
                 )
+            else:
+                if self.distributed_mode == "fsdp2":
+                    raise ValueError(f"`fsdp2` is not available for `thunder.jit`.")
```

Comment:
To make it clear to whoever sees this error that there are no plans to make it available for `thunder.jit`.
```suggestion
                    raise ValueError(f"`fsdp2` is not expected to work with `thunder.jit`. It's an option only for PyTorch.")
```